components/compositing/compositor_thread.rs

```diff
@@ -122,6 +122,10 @@ pub enum Msg {
     Status(TopLevelBrowsingContextId, Option<String>),
     /// Get Window Informations size and position
     GetClientWindow(TopLevelBrowsingContextId, IpcSender<(Size2D<u32>, Point2D<i32>)>),
+    /// Get the screen size
+    GetScreenSize(TopLevelBrowsingContextId, IpcSender<(Size2D<u32>)>),
+    /// Get the available screen size
+    GetScreenAvailSize(TopLevelBrowsingContextId, IpcSender<(Size2D<u32>)>),
```

Comment:
This isn't really an issue with this PR, but it would be nice if we could give these typed units, since I'm not sure what the unit is (device pixels?). A comment with the unit at least would be nice.

Reply:
Are you suggesting we use `Typed*` for all compositor message? Will that mean more bytes passing through IPC? or is it just a type system thing and has no runtime impact?

Reply:
Typed units do not have a runtime impact.